### ISSUE TITLE
ARMS needs to be added to data model and to templates 

### ISSUE THREAD

**Issue by kmexter** [emo-bon/observatory-profile#44]
We need to add the new ARMS logsheet metadata to the EMOBON ontology and to the ttl template
The current ARMS template logsheet is https://docs.google.com/spreadsheets/d/1gGOmCKFP3LXlTGFRXDzWe4aNb8NDp_hRIyhCrFJkb5w/edit?gid=0#gid=0

(@laurianvm can you arrange a meeting with me when I get back in August for us to do this?)

**Comment by laurianvm** [comment]
(combining previous old comments into one)

**Things to be included:**

- add predicate to Observatory class pointing to itself: emobon:isPartOfObservatory {ObsID}
    & add emobon:hasUnitID {ARMSunitID itself as the unit ID}
    & add isSpatialReplicate if the check passes

or put differently:
ObsID +ARMSUnitID (e.g. KOSTER_VH1) = an Observatory with parentObs (e.g. KOSTER) & a unitID (VH1) & info of it being a spatialreplicate (true/false, in case it passes + in case true also add BODC term (https://vocab.nerc.ac.uk/collection/P01/current/IDSPRE01/) being equal to the unitID)

- add deployment date & collection date (instead of one date atm) for a sampling event

- add predicate for crate_cover & define in the ontology

- add preservative & possibly define in ontology (look for term first)

- add predicate for samp_fraction & define in ontology

- add predicate for size_filter & define in ontology

- add boolean predicate for images_in_PlutoF & define in ontology


**Terms to include from logsheet_schema_extended.csv:**

![Image](https://github.com/user-attachments/assets/01ee2f5a-4909-451a-b369-b5418c91130e)

**Comment by kmexter** [comment]
actually no, the governance files - e.g. those in https://github.com/emo-bon/governance-crate/, but also some of those in the sequencing crate, and one or two thers - are created manually and yes some of that comes from the logsheets but quite a bit comes separately

**Comment by kmexter** [comment]
> [@laurianvm](https://github.com/laurianvm) the 'data_owner', 'data_owner_edmoid', and 'data_contact_email' columns are also in the sediment and water logsheets, in the "observatory" sheet. I think for all of them the values should be 'European Marine Biological Resource Centre', '5669', and '[emobon@embrc.eu](mailto:emobon@embrc.eu)' .

ok, I see that now. I dont remember anyone telling us that those had been added. Please raise an issue before making any changes so we know about them.

**Comment by laurianvm** [comment]
@cpavloud  can values of the ['env_package' column](https://docs.google.com/spreadsheets/d/1gGOmCKFP3LXlTGFRXDzWe4aNb8NDp_hRIyhCrFJkb5w/edit?gid=609453952#gid=609453952&range=O1) be "hard"?
it would be consistent with 'sediment' and 'water' and use of ha_id further in the spreadsheet :)

**Comment by cpavloud** [comment]
> I am not going to add these new cols to the QC checks - unless youse really think we need to check those also?

@kmexter I don't think we need to check these.

**Comment by kmexter** [comment]
who will make the changes to the logsheets?

**Comment by cpavloud** [comment]
Changes are done to all the logsheets: "arr_hq_date" has been changed to "arr_date_hq".

**Comment by laurianvm** [comment]
FYI – our plan is to close this issue by 16 November, so it would be ideal if a decision could be made beforehand. I understand everyone’s schedules are tight.

**Comment by kmexter** [comment]
> I am a bit confused as to what needs to be done still for this issue to close... (It has gotten rather long)
> 
> `1, create an ARMS checklist in ENA. Urgent I think `
> 
> This cannot be done by us. At this stage we can only use the terms we want and find meaningful.
> 

Indeed, MIxS (or ENA or GSC?) have to do it, but I thought it had to be triggered by us (via a GH issue?); or if already triggered, then pushed by us. So it is a todo for ARMS-MBON/EMO BON but not something that we have to do _in this gh space_.

> `The ARMS logsheets in google drive still have potential booboos in them that need fixing as the upload-QC code will fail. We should review all of these in the Dec meeting,`
> 
> It might too late though if we correct them in Dec, if [@laurianvm](https://github.com/laurianvm) wants things ready in two weeks.

Well, the sooner they are corrected the better, of course, I did bring this up at the previous ARMS-MBON meeting ([meeting minutes](https://docs.google.com/document/d/1MPWYwtisT8hpMLT6C_rG_-nC79eR_uayoZACcd1JPzw/edit?tab=t.0) and it is really for the stations to do the corrections (tho Ioulia and you could also, as some of the corrections are just "remove these rows".)  I did not raise an issue on these points, Should I?

There are several other ARMS issues to solve (https://github.com/search?q=org%3Aemo-bon+ARMS&type=issues), some are on me and some are on others. It is these I thought we should discuss in Dec. The most important of those wrt getting the QC pipeline working is https://github.com/emo-bon/data-quality-control-action/issues/5. I will write down in there what I think the QC rules should be for these logsheets and then @bulricht can implement that. 

> ` template updates may be required to handle or process the values correctly. If that’s the case, please open a new issue so it can be addressed separately.`
> 
> I am confused as to why we will need to open new issues... If you mean that this issue was only to add ARMS in the workflow, then (since this is done) we can close it and move the discussion about the logsheets elsewhere.

Yes, once we can run the upload-QC-create triples, if something fails we then deal with that by raising new issues (as opposed to trying to sort them out in this one). So yes, I will close this issue now
